source/appendices/e-audit-requirements-and-tests/06-usage-tests.rst

```diff
@@ -0,0 +1,67 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2023 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+E.6 Audit Tests for Investigations and Requests for Other Data_Types
+--------------------------------------------------------------------
+
+This section of the appendix outlines tests that MUST be run during audits for any platform delivering any Data_Type that is not Book or Reference_Work content. For report providers operating multiple platforms including one or more with non-Book or Reference_Work content, the audit scope as defined in :numref:`audit` MUST include one of these platforms and be subject to the audit tests outlined here. Note that these tests refer to 'item' for ease of reading, without specifying Data_Type.
+
+These tests apply to the Platform Report, Database Report, Title Report and Item Report. The PR_P1, DR_D1, TR_J1, TR_J3, TR_J4 and IR_A1 Standard Views of the COUNTER Reports will be deemed COUNTER-compliant if the metrics and Data_Types match those in the relevant COUNTER Report, with the appropriate aggregation.
+
+These audit tests specify minimum numbers of items to be tested. Where a platform has fewer than the required number of items, the auditor MUST make at least one request for each item on the platform, and where necessary undertake duplicate actions to reach the required threshold.
+
+Audit test requirements vary depending on the set up of the platform, as indicated by the Options within the tests below.
+
+
+E.6.1 Access Types
+""""""""""""""""""
+
+Where possible, 50% of items requested SHOULD be via browsing the platform and 50% via searching the platform. If either browsing or searching is not possible, all items can be requested via the only available option.
+
+**Option 1**: Report provider offers items under just one Access_Type.
+
+The auditor MUST request 100 items.
+
+This MUST result in 100 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests with the appropriate Access_Type.
+
+**Option 2**: Report provider offers items under two Access_Types.
+
+The auditor MUST request 50 items with each Access_Type.
+
+This MUST result in 50 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests with each Access_Type.
+
+The Access_Type combinations might be: Controlled plus Open, Controlled plus Free_To_Read, or Open plus Free_To_Read.
+
+**Option 3**: Report provider offers Controlled, Open and Free_To_Read items.
```

Comment:
For consistency "under all three Access_Types (Con-trolled, Open and Free_To_Read)"?